tox.ini

```diff
@@ -22,6 +22,3 @@ deps = flake8
 
 [testenv:flake8]
 commands = flake8 zigzag setup.py tests
-
-[pytest]
-python_files = tests/test_*.py
```

Comment:
I had to remove this because otherwise pytest would always use this setting regardless of what the user provided via the command-line.